bears/python/PyUnusedCodeBear.py

```diff
@@ -15,7 +15,7 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
```

Comment:
Thank you will make changes. I have a doubt. Earlier was remove_all_unused_imports = False getting overridden by remove_all_unused_imports=true present in the .coafile?